db/app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { getCategories } = require("../controllers/categoriesController.js");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
Please remove this express.json as we don't need it just yet. We can add it when you deal with post and patch requests :)